### ISSUE TITLE
Create Chrome Web Store Promotional Images

### ISSUE THREAD

**Issue by jasonLaster** [marionettejs/marionette.inspector#219]
We used to have promo images, then google decided they were bad. Lets make new ones!

![](http://f.cl.ly/items/060T3K2U0V3124100C0H/Image%202015-01-15%20at%2010.28.38%20AM.png)

@nerdpruitt, @jdaudier - looking towards you guys. Any interest?

**Comment by jdaudier** [comment]
I'd like to help with this one but just need some ideas on what we should capture this time around.  Should we just do screenshots from a smaller app so there's less views / code?

**Guidelines:**
https://developer.chrome.com/webstore/images?hl=en#promo

**Old screenshots that Google rejected for reference:**
![img](https://s3.amazonaws.com/f.cl.ly/items/0p3L093B1D0v3t380c3T/Image%202015-01-15%20at%2010.49.32%20AM.png)

![img](https://s3.amazonaws.com/f.cl.ly/items/3e1j3x1y1j2u2E1b2b2b/Image%202015-01-15%20at%2010.49.23%20AM.png)

![img](https://s3.amazonaws.com/f.cl.ly/items/2f253b063r3Z3L2I1z1Z/Image%202015-01-15%20at%2010.49.14%20AM.png)

**Comment by jasonLaster** [comment]
Hmm. Maybe an expanded view of the logos? Something representational of the inspector w/o any text?

**Comment by samccone** [comment]
yeah it is crazy, since the inspector is supposed to look technical... perhaps we can hook someone from google dev relations into this thread....

**Comment by jdaudier** [comment]
Ah so i did some research. I got it now.  It's different than screenshots.

**Details**
![1](https://cloud.githubusercontent.com/assets/1666031/5764309/ef523790-9cbe-11e4-890e-afe0d805d92e.png)

![2](https://cloud.githubusercontent.com/assets/1666031/5764311/f32d7140-9cbe-11e4-90e1-3a1e5ce266cf.png)

![3](https://cloud.githubusercontent.com/assets/1666031/5764314/f662a5d8-9cbe-11e4-88ac-f5f765b53c9c.png)

![5](https://cloud.githubusercontent.com/assets/1666031/5764316/f92d85d0-9cbe-11e4-90dd-e0e6b682a882.png)

**Comment by jdaudier** [comment]
So @jasonLaster I think you should choose one of the inspector logo designs, so I can use it for these promo images.
